lang/cy.json

```diff
@@ -297,17 +297,28 @@
     "submittedYourLpa": "Rydych wedi cyflwyno eich LPA",
     "whatHappensNext": "Beth fydd yn digwydd nesaf",
     "weWillContactYourCertificateProvider": "Byddwn yn cysylltu â’ch darparwr tystysgrif, {{.FirstNames}}, ac yn gofyn iddynt gadarnhau eu hunaniaeth.",
+    "opgWillContactYourCertificateProvider": "Welsh {{.FullName}}",
     "weWillThenContactAttorneys": {
         "one": "Yna byddwn yn cysylltu â {{.FullNames}} ac yn gofyn iddynt lofnodi eich LPA fel eich atwrnai.",
         "two": "Yna byddwn yn cysylltu â {{.FullNames}} ac yn gofyn iddynt lofnodi eich LPA fel eich atwrneiod.",
         "few": "Yna byddwn yn cysylltu â {{.FullNames}} ac yn gofyn iddynt lofnodi eich LPA fel eich atwrneiod.",
         "many": "Yna byddwn yn cysylltu â {{.FullNames}} ac yn gofyn iddynt lofnodi eich LPA fel eich atwrneiod.",
         "other": "Yna byddwn yn cysylltu â {{.FullNames}} ac yn gofyn iddynt lofnodi eich LPA fel eich atwrneiod."
     },
+    "opgWillContactAttorneys": {
+        "one": "Welsh {{.FullNames}}",
+        "two": "Welsh {{.FullNames}}",
+        "few": "Welsh {{.FullNames}}",
+        "many": "Welsh {{.FullNames}}",
+        "other": "Welsh {{.FullNames}}"
+    },
     "onceEveryoneHasSigned": "Unwaith y bydd pawb wedi llofnodi a phan fydd eich LPA wedi’n cyrraedd, byddwn yn hysbysu unrhyw bobl y byddwch yn dewis rhoi gwybod iddynt. Bydd cyfnod aros o 4 wythnos yn dechrau. Mae hyn yn ofynnol yn ôl y gyfraith, er mwyn caniatáu i unrhyw un sydd â gwrthwynebiad i’w codi.",
+    "onceEveryoneHasSignedOpgWill": "Welsh",
```

Comment:
Should this (and other mid-key instances) be capitalised for consistency?

Reply:
I don't think we've been consistent with the keys anyway, the only one I spotted allcaps was `theOPG`, which isn't even used...